keymapviz/keyboards/kaishi65/__init__.py

```diff
@@ -0,0 +1,17 @@
+keymap_keyword = '(?:LAYOUT_65_ansi)'
+layout_editor_json = 'keyboards/kaishi65/layout_editor.json'
+
+ascii_art = '''
+    /* ,----------------------------------------------------------------.
+     * |{} |{} | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
+     * |----------------------------------------------------------------|
+     * | {}  | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}  | {} |
+     * |----------------------------------------------------------------|
+     * |  {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}   | {} |
+     * |----------------------------------------------------------------|
+     * |   {}   | {}| {}| {}| {}| {}| {}| {}| {}| {}| {}|  {}  | {}| {} |
+     * |----------------------------------------------------------------|
+     * | {} | {} | {} |          {}           | {}| {}| {} | {}| {}| {} |
+     * `----------------------------------------------------------------'                     
+     * generated by [keymapviz]
```

Comment:
Please add c language block comment end `*/`.
```suggestion
     * generated by [keymapviz]
     */
```